src/protocols/at-ps2/mouse_interface.c

```
@@ -132,226 +145,246 @@ int8_t get_z_movement(uint8_t pos) {
-          mouse_config_sequence = 0;
-          mouse_state = INITIALISED;
-          LOG_INFO("Mouse Initialisation Complete\n");
-        } else {
-          mouse_command_handler(config_sequence[mouse_config_sequence]);
-          mouse_config_sequence++;
+    static uint8_t mouse_type_detect_sequence = 0;
+    static uint8_t mouse_max_packets          = 0;
+
+    switch (mouse_state) {
+        case UNINITIALISED:
+            switch (data_byte) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)